<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_admin_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>416</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_admin_acount">
    <attribute name="title">
     <string>开户</string>
    </attribute>
    <widget class="QLabel" name="label_op_acount_user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_op_amount_money">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>存款金额：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_op_acount_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_op_acount_newpassword">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_op_acount_user">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_op_amount_money">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_op_acount_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_acount_newpassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_op_ok">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_op_back">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>330</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_admin_close_acount">
    <attribute name="title">
     <string>销户</string>
    </attribute>
    <widget class="QLabel" name="label_close_user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_close_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close_ok">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_close_user">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_close_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close_back">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_admin_update_password">
    <attribute name="title">
     <string>修改密码</string>
    </attribute>
    <widget class="QLabel" name="label_alter_user">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_alter_oldpassword">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>原密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_alter_newpassword">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_alter_okpassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_alter_user">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_alter_oldpassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_alter_newpassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_alter_okpassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_alter_ok">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_alter_back">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
